schemas/resources/schemas/service-entry/v1.0.0/search.md

```diff
@@ -93,7 +93,7 @@ The JSON response includes the following fields.
 __Example__
 
 ```
-    curl -H "Cmr-Pretty: true" "%CMR-ENDPOINT%/service-entries.json?name="service-entry-name"
+    curl -H "Cmr-Pretty: true" "%CMR-ENDPOINT%/service-entries.json?name="Serviceentry-name-v1"
```

Comment:
why is this example without -?

Reply:
To make sure it matches the response but, I can set them all to use the "-" characters instead if that is preferred, that way would be congruent with the actual concept name

Reply:
I see, it's just data

Reply:
Right, perhaps it's even better so not as to imply that there is some kind of required link between the name and the concept-type when it's really just a string.